sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -4718,7 +4730,6 @@ _ZNK4sycl3_V18platform11get_backendEv
 _ZNK4sycl3_V18platform11get_devicesENS0_4info11device_typeE
 _ZNK4sycl3_V18platform13has_extensionERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE
 _ZNK4sycl3_V18platform30ext_oneapi_get_default_contextEv
-_ZN4sycl3_V16detail33enable_ext_oneapi_default_contextEb
```

Comment:
Why it is removed? Is it ABI break between the releases?

Reply:
This symbol is moved to another line because  the script was used to generate this file.